Improvements to Castle

acerspyro opened the pull request.

acerspyro committed:
Castle update

Fixed bug where a piece of geom could be found on top of one of the outer windows
Improved torches
Improved roof
Improved map skirt
Changed lighting
Improved ladders

acerspyro committed:
Change map path

qreeves reviewed (changes requested):
Too much negative space at the top of the map image. It also looks a little washed out?

acerspyro commented:
My screen's color profile is way off, so I wouldn't know. I'll try it on my Chromebook later.

acerspyro committed:
Requested changes, and more

Fade effect at the bottom of the map, culled textures, entities and sounds, added myself as a map author, reverted ambient lighting, fixed map shot.

acerspyro committed:
Gotta be humble

acerspyro committed:
Revert previous commit, I don't feel humble today.
